packages/api/src/command/medical/tcm-encounter/get-tcm-encounters.ts

```diff
@@ -111,7 +111,11 @@ export async function getTcmEncounters({
           : ""
       }
       ${coding === "cardiac" ? ` AND tcm_encounter.has_cardiac_code = true` : ""}
-      ${encounterClass ? ` AND tcm_encounter.class = :encounterClass` : ""}
+      ${
+        Array.isArray(encounterClass) && encounterClass.length
```

Comment:
Why do we need this isArray? We know its either of type array or its undefined.

encounterClass !== undefined && encounterClass.length > 0 is more representative of what we're actually needing to do based on the types